lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -35,9 +35,7 @@ class EstimatedInputLatency extends Audit {
       name: 'estimated-input-latency',
       description: 'Estimated Input Latency',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
-      helpText: 'Primarily a measure of the main thread\'s activity. ' +
-      'There is a 90% probabililty a user would encounter input latency of this amount or less. ' +
-      '10% of touches can expect additional latency.',
+      helpText: 'If your app takes more than 100ms to respond to user input, users perceive the app as laggy. <a href="https://developers.google.com/web/tools/lighthouse/audits/estimated-input-latency" rel="noopener noreferrer" target="_blank">Learn more</a>.',
```

Comment:
> I added this to the audit's doc: google/WebFundamentals#3887

> Here's my general opinion on help text content: when a doc exists for an audit, then implementation details like this should go in the "what the audit tests for" section of the doc. Situations like this are why I created that section for the docs, and I want to encourage people to get used to going to the docs for this type of info. The help text should only summarize the audit, and why the user cares about it. When a doc doesn't exist, then we can use the help text as a stop gap.

Yeah, I generally agree, I guess my issue is that there's no context for the number without the old `helpText`. "Latency of which input at what time?", etc. Maybe the `displayValue` of the result should incorporate some of that somehow.

We also have the disconnect here between the 50ms goal but the 100ms in the `helpText`. This is also explained in the docs, but it's just confusing without clicking through. I wonder if there's a way to re-word it to use 50ms in the `helpText` instead of 100.